### DISCUSSION TITLE
cloud-is-broken

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#24] [Giscus]
# cloud-is-broken

Introduction: Recently I was chatting with an investor about the market and they asked: What do you see coming for the world of Data/Cloud/Compute Infrastructure? Is there some new dataframe librar...

https://matthewrocklin.com/cloud-is-broken.html

**Comment by samuelsinayoko** [comment]
You are spot on. It’s hard to love a lot of these cloud services at the moment.
UX or DevX are critical. I think this is also a key reason for Python’s success as a language (readable and accessible, yet powerful) over competitors, despite the packaging pains. Docker’s success is also down to  UX I would argue.

**Comment by carloshkayser** [comment]
Great article!
